### ISSUE TITLE
Description markdown code blocks labeled `jsx` don't get formatted 

### ISSUE THREAD

**Issue by jaydenseric** [hosseinmd/prettier-plugin-jsdoc#137]
In `demo.mjs`, if you have this content:

```js
/**
 * ```js
 * let   a
 * ```
 * 
 * ```jsx
 * let   a
 * ```
 */
```

After formatting the content formats to this:

```js
/**
 * ```js
 * let a;
 * ```
 *
 * ```jsx
 * let   a
 * ```
 */
```

As you can see, the JSDoc description markdown code block labeled `js` gets formatted but the one labeled `jsx` doesn’t.

If you try that same markdown in a `demo.md` file, after formatting both the `js` and `jsx` labeled code blocks are correctly formatted by Prettier, proving that this Prettier plugin is somehow not recognizing all the labels that Prettier can normally format.

**Comment by jaydenseric** [comment]
Is it to do with this code missing a few entries?

https://github.com/hosseinmd/prettier-plugin-jsdoc/blob/5474d66c7b8b8117d6a2acfa34fcee0d32c2059e/src/descriptionFormatter.ts#L15-L21

There are actually a lot of other code block labels like `css` that are not supported by this plugin yet, but supported by normal Prettier in markdown.

**Comment by hosseinmd** [comment]
Yes that is here.

**Comment by jaydenseric** [comment]
Is there a reason Prettier can't be used to format the whole descriptions as if they are markdown, and let prettier handle the formatting of the bullet lists, code blocks etc. itself however it normally does?

**Comment by hosseinmd** [comment]
We tested before but did not get the right result.

**Comment by hosseinmd** [comment]
fixed v0.3.27
https://github.com/hosseinmd/prettier-plugin-jsdoc/commit/67274804347e6c19b624c4b6ab125d818a7acf2b

**Comment by jaydenseric** [comment]
@hosseinmd thanks for the speedy fixes, you're doing a champion job 🙌
